### DISCUSSION TITLE
Module not found

### DISCUSSION THREAD

**Discussion by FelipeGerolomo** [outline/rich-markdown-editor#385] [Q&A]
I got this error when i installed

./node_modules/outline-icons/lib/components/CheckboxIcon.js

Module not found: Can't resolve 'core-js/modules/es.array.index-of.js' in 'C:\Projetos\LEXP\studentcare-web\node_modules\outline-icons\lib\components'

**Comment by dave-db** [comment]
I hit the same issue, installing `core-js@3` as a dependency took care of it, but I'm not sure if it's deliberate or not.

**Reply by tommoor** [reply]
This should be resolved in `v11.3.1`, it's hard for me to know for sure because it works already in all environments I'm using so would be good to get the 👍

**Reply by dave-db** [reply]
Looks good! Removed the `core-js` dependency, all was still well. Thanks for the quick fix 🥳
